content/speakers/tilde-ann-thurium.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Tilde Ann Thurium",
+  "headshot": "/images/tilde-ann-thurium.jpg",
+  "bio": "Tilde Ann Thurium is an artist, engiqueer, and activist. In previous lives, they have done stints as a florist, a security guard, and a Human Resources wench. In their spare time, they illustrate data structures and algorithms with acrylic paint.",
+  "role": "Developer Evangelist",
+  "company": "Twilio",
+  "pronouns": "they/them",
+  "social_links": [
+    { "github": "annthurium" },
```

Comment:
so the social links should just be the usernames? not the full links, for all of them? 

what about "website"?

Reply:
You're right, for the social links — I was figuring since twitter/github/linkedin will always be the same we could just store handles. But do you think it’s too confusing?